app/models/chargeback_rate_detail.rb

```diff
@@ -2,7 +2,8 @@ class ChargebackRateDetail < ActiveRecord::Base
   belongs_to :chargeback_rate
   belongs_to :detail_measure, :class_name => "ChargebackRateDetailMeasure", :foreign_key => :chargeback_rate_detail_measure_id
 
-  validates_numericality_of :rate
+  validates :rate, :numericality => true
```

Comment:
TBH I'm not sure of the consensus here. I'm for testing it, but would defer to someone else on that decision.